Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Extensions.cs

```diff
@@ -0,0 +1,288 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I'd call this `.Helpers` over .Extensions as extensions have a specific connotation around adding methods to an existing type.

Reply:
Yes, I did.